Added close button callout for android (#649)
RevenueCat/revenuecat-docs: docs_source/Tools/paywalls/displaying-paywalls.md

]
[/block]

> ## iOS Close Button

Reviewer: This is already inside of the iOS section

Reviewer: ```suggestion
## Close Button
```

Reviewer: #676

]
[/block]

> ## Android Close Button

Reviewer: Ditto no need to mention Android here

Author: Would like me to just take it out of the iOS section then and make a section that encompasses both?

Reviewer: No, I'd leave this on each of the platforms.

Reviewer: Should we rename this as
```suggestion
## Close Button
```

Reviewer: #676

> Paywalls displayed with `Paywall` will have a close button on the presented sheet.
> However, a `PaywallDialog` will not have a close button by default. This gives you full control over how to navigate to and from your `PaywallDialog`. You can push this to the existing navigation stack or display it in a sheet with a custom dismiss button using Android's navigation components and fragments.

Reviewer 2: ```suggestion
Paywalls displayed with `PaywallDialog` will have a close button on the presented sheet.
However, a `Paywall` will not have a close button by default. This gives you full control over how to navigate to and from your `Paywall` composable. You can push this to the existing navigation stack or display it in a sheet with a custom dismiss button using Android's navigation components and fragments.
```
I believe this is the other way around. Paywalls displayed using `Paywall` will NOT have a close button, since they won't be presented as a sheet, the developer is in charge of positioning the composable.

However, if they use `PaywallDialog`, we will show the close button by default, since we will be in charge of showing the paywall as a modal.

> If desired, you can pass `shoulDisplayDismissButton: true` when creating `PaywallView` to display a close button automatically.

Reviewer 2: ```suggestion
If desired, you can use `PaywallOptions.Builder`'s `setShouldDisplayDismissButton(true)` method when creating a `Paywall` to display a close button automatically.
```
